source/features/create-release-shortcut.tsx

```diff
@@ -22,4 +26,11 @@ void features.add({
 		pageDetect.isReleasesOrTags
 	],
 	init
+}, {
```

Comment:
Missing `shortcuts`

Reply:
I was treating it like a fix.

Will add